src/java/org/apache/cassandra/service/snapshot/SnapshotManagerMBean.java

```diff
@@ -78,14 +78,24 @@ public interface SnapshotManagerMBean
     long getTrueSnapshotsSize(String keyspace);
 
     /**
-     * Get the true size take by all snapshots in given keyspace and table.
+     * Get the true size taken by all snapshots in given keyspace and table.
      *
      * @param keyspace keyspace to get true size of all snapshots of
      * @param table table in a keyspace to get true size of all snapshots of
      * @return true size of all snapshots in given keyspace and table
      */
     long getTrueSnapshotsSize(String keyspace, String table);
 
+    /**
+     * Get the true size of a snapshot in given keyspace and table.
+     *
+     * @param keyspace keyspace to get true size of all snapshots of
+     * @param table table in a keyspace to get true size of all snapshots of
+     * @param snapshotName name of snapshot in given keyspace and table to get true size of
+     * @return true size of all snapshots in given keyspace and table
+     */
+    long getTrueSnapshotsSize(String keyspace, String table, String snapshotName);
```

Comment:
I noticed that a user does not have any way to get true disk space used for a particular snapshot except reading it from snapshot details which is overkill so I added this simple utility method for that.